csv-bulk-signature/src/safety-checks.ts

```diff
@@ -26,20 +26,24 @@ export async function checkFileCountWithCsvCount(
   }
 }
 
-export async function hasContactsFile() {
-  return await fs.promises
-    .access(CSV_FILE)
-    .then(() => {
-      return true;
-    })
+export async function isPath(path: string, catchCallback?: () => void) {
+  return await fs
+    .stat(path)
+    .then(() => true)
     .catch(() => {
-      console.log("🔴 contacts.csv file does not exist");
+      if (catchCallback) catchCallback();
       return false;
     });
 }
 
+export async function hasContactsFile() {
+  return await isPath(CSV_FILE, () =>
+    console.log("🔴 contacts.csv file does not exist")
+  );
+}
```

Comment:
- make a isPath function to check file + add call back for errors
- refactor hasContacts to just supply the error needed if contact.csv doesn't exist